packages/ui5-test-writer/templates/freestyle/webapp/test/1.120.0/unit/unitTests.qunit.ts

```diff
@@ -3,8 +3,8 @@ QUnit.config.autostart = false;
 
 // import all your QUnit tests here
 void Promise.all([
-	import("sap/ui/core/Core"), // Required to wait until Core has booted to start the QUnit tests 
-import("unit/controller/<%= viewName %>Page.controller")
-]).then(([{ default: Core }]) => {
+	import("sap/ui/core/Core"), // Required to wait until Core has booted to start the QUnit tests
+import("unit/controller/{{simple.parameters.ViewName.value}}Page.controller"),
```

Comment:
```suggestion
import("unit/controller/<%= viewName %>Page.controller")
```
This is expected ejs syntax, right?

Reply:
Thanks for pointing this...